TA-cyences/metadata/default.meta

```diff
@@ -31,6 +31,15 @@ export = system
 export = system
 
 
+### ALERT ACTIONS
+
+[alert_actions]
+export = system
```

Comment:
Have we tested below:
When we open Cyences' alert from other App and make changes to alert action parameters, will it update the parameter under etc/apps/cyences_app_for_splunk/local/savedsearches.conf?

Reply:
yes. it is working as expected